pkg/component/networking/vpn/seedserver/seedserver.go

```diff
@@ -954,19 +944,17 @@ func (v *vpnSeedServer) deployVPA(ctx context.Context) error {
 		vpa.Spec.ResourcePolicy = &vpaautoscalingv1.PodResourcePolicy{
 			ContainerPolicies: []vpaautoscalingv1.ContainerResourcePolicy{
 				{
-					ContainerName: deploymentName,
-					MinAllowed: corev1.ResourceList{
-						corev1.ResourceMemory: resource.MustParse("20Mi"),
-					},
+					ContainerName:    deploymentName,
 					ControlledValues: &controlledValues,
 				},
 				{
-					ContainerName: envoyProxyContainerName,
-					MinAllowed: corev1.ResourceList{
-						corev1.ResourceMemory: resource.MustParse("100Mi"),
-					},
+					ContainerName:    envoyProxyContainerName,
 					ControlledValues: &controlledValues,
```

Comment:
Nit: Not introduced with this PR but we could use directly:
```suggestion
					ControlledValues: ptr.To(vpaautoscalingv1.ContainerControlledValuesRequestsOnly),
```

This is a left-over from the times when there are no generic pointer pkg such as `k8s.io/utils/ptr` (the old non-generic pkg was `k8s.io/utils/pointer`).